nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestExecuteSQLRecord.java

```diff
@@ -106,14 +98,29 @@ public class TestExecuteSQLRecord {
             + " where PER.ID < ? AND REL.ID < ?";
 
 
-    @BeforeClass
+    @BeforeAll
     public static void setupClass() {
+        System.setProperty("org.slf4j.simpleLogger.defaultLogLevel", "info");
+        System.setProperty("org.slf4j.simpleLogger.showDateTime", "true");
+        System.setProperty("org.slf4j.simpleLogger.log.nifi.io.nio", "debug");
+        System.setProperty("org.slf4j.simpleLogger.log.nifi.processors.standard.ExecuteSQLRecord", "debug");
+        System.setProperty("org.slf4j.simpleLogger.log.nifi.processors.standard.TestExecuteSQLRecord", "debug");
```

Comment:
SLF4J logging system properties should be removed. The Derby logging property can remain.

Reply:
Yep removing everything but the Derby logging